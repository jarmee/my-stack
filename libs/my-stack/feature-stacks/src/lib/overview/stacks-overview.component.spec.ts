import { ComponentFixture, TestBed } from '@angular/core/testing';

import { StacksOverviewComponent } from './stacks-overview.component';

describe('StacksOverviewComponent', () => {
  let fixture: ComponentFixture<StacksOverviewComponent>;
  let component: StacksOverviewComponent;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports: [StacksOverviewComponent],
    }).compileComponents();
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(StacksOverviewComponent);
    component = fixture.componentInstance;
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
