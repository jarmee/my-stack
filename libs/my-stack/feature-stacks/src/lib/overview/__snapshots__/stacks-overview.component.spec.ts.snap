// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StacksOverviewComponent should match snapshot 1`] = `
<ng-component
  __ngContext__={[Function Number]}
  _service={[Function StacksService]}
  stacks$={[Function Observable]}
>
  <div
    class="px-4"
  >
    <div
      class="flex items-end"
    >
      <button
        class="mb-3 mdc-button mat-mdc-button"
        mat-button=""
        routerlink="create"
        tabindex="0"
      >
        <span
          class="mat-mdc-button-persistent-ripple"
        />
        <mat-icon
          aria-hidden="true"
          class="mat-icon notranslate"
          role="img"
        >
          add
        </mat-icon>
        <span
          class="mdc-button__label"
        />
        <span
          class="mat-mdc-focus-indicator"
        />
        <span
          class="mat-ripple mat-mdc-button-ripple"
          matripple=""
        />
        <span
          class="mat-mdc-button-touch-target"
        />
      </button>
      <h3>
        Stacks
      </h3>
    </div>
    <mat-divider
      class="mat-divider"
      role="separator"
    />
    <div
      class="grid xl:grid-cols-8 lg:grid-cols-6 md:grid-cols-4 sm:grid-cols-3 grid-cols-2 gap-4 mt-3 h-100"
    >
    </div>
  </div>
</ng-component>
`;
